README.md

```diff
@@ -1,493 +1,69 @@
-helm inspect values charts/ti-key-release-2-X.X.X.tgz > config.yaml
-# modify config.yaml
-helm install -i --values=config.yaml tsi-install charts/ti-key-release-2-X.X.X.tgz
-# or upgrade existing deployment
-helm upgrade -i --values=config.yaml tsi-install charts/ti-key-release-2-X.X.X.tgz
-```
-
-### Boostrapping - CI/CD pipeline
-The bootstrapping process is shown in details under the [Vault demo](examples/vault/README.md)
-
-## Run Demo
-For next steps, review [demo](examples/README.md) examples.
-
-## Sample JWT claims
-Once the TSI environment is operational, the application container will have
-access to JWT Token. The token can be inspected in the JWT [Debugger](https://jwt.io/) in Encoded window.
-
-Sample JWT Claims:
-```json
-{
-  "cluster-name": "ti_demo",
-  "region": "us-south",
-  "exp": 1557170306,
-  "iat": 1557170276,
-  "images": "f36b6d491e0a62cb704aea74d65fabf1f7130832e9f32d0771de1d7c727a79cc",
-  "images-names": "tsidentity/myubuntu@sha256:5b224e11f0e8daf35deb9aebc86218f1c444d2b88f89c57420a61b1b3c24584c",
-  "iss": "wsched@us.ibm.com",
-  "machineid": "fa967df1a948495596ad6ba5f665f340",
-  "namespace": "trusted-identity",
-  "pod": "myubuntu-698b749889-vvgts",
-  "sub": "wsched@us.ibm.com"
-}
-```
-
-### Cleanup
-Remove all the resources created for Trusted Identity
-```console
-./utils/cleanup.sh
-```
-To start a fresh deployment, make sure to run `./utils/init-namespace.sh` again after the cleanup.
-
-## Security Considerations
-To improve security measures, we have decided to move all application containers
-to a different namespace and keep all the TSI framework as _protected namespace_
-that cannot host application and other containers (since version 1.4)
+This Universal Workload Identity project
+(also known as Trusted Service Identity)
+provides a deployment and an orchestration layer to support
+CNCF community initiatives
+[Tornjak](https://github.com/spiffe/tornjak)
+and [SPIRE](https://github.com/spiffe/spire).
+
+*Notice:*
+* For all the original *Trusted Workload Identity* project,
+that was preceding the SPIRE and Tornjak integration,
+and focusing on keys and credentials management, and preventing
+access to secrets by untrusted administrator,
+please visit our [main-no-spire](../../tree/main-no-spire/) branch.
+
+## Introduction
+Here is the stack that represents the layers of
+the Universal Workload Identity project.
+Most of the components are part of the [CNCF](https://www.cncf.io/)
+(Cloud Native Computing Fundation) initiative.
+
+![technology-stack](docs/imgs/CNCF.stack.jpg)
+
+Starting from the bottom, we support any Kubernetes Platform, whether this is
+a native Kubernetes or OpenShift.
+
+Then we have SPIFFE, which defines the identity format and specifies how
+workloads can securely obtain identity.
+
+Then we have SPIRE, which implements SPIFFE,
+and it provides the zero trust attestation of workloads and infrastructure.
+SPIFFE is responsible for issuing and rotating of x509 certificates or
```

Comment:
SPIFFE here is ok, but very minor nit, may be clearer here if we say SPIRE. Because SPIFFE conceptually does it but SPIRE conceptually and literally does it